tools/cleanup_template/pubspec.yaml

```diff
@@ -6,15 +6,14 @@ environment:
   sdk: ^3.3.0
 
 dependencies:
-  file: ^7.0.0
+  file: ^6.1.4
```

Comment:
@Yamasaki-pan961 
`yumemi_lints` 2.0.0 は Dart 2.17.0 以降をサポートしている関係上、 `yumemi_lints` パッケージ内でも `file: ^6.1.4` の指定をしているため、 `file` のバージョンは 6.1.4 以上 7.0.0 未満を指定しなければ次のエラーが発生してしまいます。

```
An unexpected error occurred.
Consider creating an issue on GitHub Issue.

Because no versions of yumemi_lints match >2.0.0 <3.0.0 and yumemi_lints 2.0.0 depends on file ^6.1.4, yumemi_lints ^2.0.0 requires file ^6.1.4.
So, because cleanup_template depends on both file ^7.0.0 and yumemi_lints ^2.0.0, version solving failed.


You can try one of the following suggestions to make the pubspec resolve:
* Consider downgrading your constraint on yumemi_lints: dart pub add dev:yumemi_lints:^1.7.0
* Consider downgrading your constraint on file: dart pub add file:^6.1.4
```

今回は `yumemi_lints` 2.0.0 を優先したため、このような修正をする必要がありました。

Reply:
理解しました！！
ご説明ありがとうございます！！